.vsts-ci.yml

```diff
@@ -130,23 +129,83 @@ phases:
       name: MoveLibgpiod
       displayName: Move Libgpiod into crossrootfs
       condition: succeeded()
-    - script: ROOTFS_DIR=/crossrootfs/arm eng/common/cibuild.sh
+    - script: ROOTFS_DIR=/crossrootfs/arm ./build.sh --ci
         --configuration $(_BuildConfig)
         --prepareMachine
         /p:BuildNative=true
       name: Build
       displayName: Build
       condition: succeeded()
+    - task: PublishBuildArtifacts@1
+      displayName: Publish Linux native assets
+      inputs:
+        pathToPublish: $(Build.SourcesDirectory)/artifacts/bin/Native
+        artifactName: Native
+        artifactType: container
+      condition: eq(variables['_BuildConfig'], 'Release')
     variables:
       _HelixBuildConfig: $(_BuildConfig)
 
+- template: /eng/common/templates/phases/base.yml
+  parameters:
+    dependsOn:
+    - Windows_NT
+    - OSX
+    - Linux
+    agentOs: Windows_NT
+    name: BuildPackages
+    enableTelemetry: $(_enableTelemetry)
+    queue:
+      ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        name: dotnet-internal-temp
+      ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+        name: dotnet-external-temp
+      parallel: 99
+      matrix:
+        release_configuration:
+          _BuildConfig: Release
+    ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+      enableMicrobuild: true
+    steps:
+    - task: DownloadBuildArtifacts@0
+      displayName: Download Built Managed Assets
+      inputs:
+        artifactName: artifacts
+        downloadPath: $(Build.SourcesDirectory)
```

Comment:
should this have a artifacts in the path, or is the task appending the artifactName?  (same below)

Reply:
yup it is, this will create a folder called artifacts under the sources directory, which is what I want so that it respects the output as it was where the assets where generated.